### PR TITLE
add : korean translation to doc

### PR DESCRIPTION
**Pull request by codevilot**
I think it's a paradigm shifting project.
I just saw a post related to Pyscript and I really liked it, so I translated it into a Korean version and uploaded it. 
I hope more and more people will use Pyscript.

Thank you for making a good project and for reading it.

### CODE REVIEW

**Comment by weensy** [comment]
Thank you for your work.
However, some parts of this are not smooth like the translator.
Can I make a little suggestion?

File: docs/doc_kr/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+예제보기 [the pyscript folder](pyscriptjs).
+
+### Longer Version
+PyScript는 사용자가 파이썬으로 정교한 브라우저 애플리케이션을 만들 수 있게 하는 여러 오픈소스 기술을 결합을 목표로 하는 메타 프로젝트입니다.
+PyScript는 브라우저에서 DOM이 작동하는 방식과 정교하게 결합하고 웹과 파이썬 개발자 모두에게 자연스럽게 파이썬 로직을 추가할 수 있습니다.
+
+
+## PyScript 사용하기
+
+PyScript을 사용하기 위해서, html 페이지에 올바른 pyscript 파일을 import 해야 합니다:
```
Comment (weensy) [inline docs/doc_kr/README.md:19]:
```suggestion
PyScript를 사용 하려면, 다음과 같이 html 페이지에 적절한 pyscript 파일들을 import 해야 합니다.
```

File: docs/doc_kr/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```html
+<link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
+<script defer src="https://pyscript.net/alpha/pyscript.js"></script>
+```
+이제 PyScript 컴포넌트를 html page에 사용할 수 있습니다. PyScript는 현재 다음과 같은 요소(elements)들을 구현하고 있다.
```
Comment (weensy) [inline docs/doc_kr/README.md:24]: - Giving unity to the translation of repeated words.
- Giving unity to the use of honorifics.
```suggestion
이제 PyScript 컴포넌트들을 html 페이지에 사용할 수 있습니다. PyScript는 현재 다음과 같은 요소(elements)들을 구현하고 있습니다.
```

File: docs/doc_kr/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# PyScript
+
+## PyScript란?
+
+### Summary
+PyScript는 Scratch, JSFiddle 및 기타 "사용하기 쉬운" 프로그래밍 프레임워크에 대한 파이썬의 대안이며, 누구나 흥미롭고 상호작용하는 애플리케이션을 친근하고 나만의 방식대로 웹에 만드는 것을 목표로 합니다.
+
+시작하려면 [getting started tutorial](docs/tutorials/getting-started.md).
```
Comment (weensy) [inline docs/doc_kr/README.md:8]:
```suggestion
시작하려면 [시작 튜토리얼](docs/tutorials/getting-started.md)을 참조하세요.
```

File: docs/doc_kr/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+## Resources
+
+* [Discussion board](https://community.anaconda.cloud/c/tech-topics/pyscript)
+* [Home Page](https://pyscript.net/)
+* [Blog Post](https://engineering.anaconda.com/2022/04/welcome-pyscript.html)
+
+## Notes
+
+* 이것은 매우 실험적인 프로젝트이므로, 프로젝트가 깨질 수 있습니다!
+* PyScript는 현재 크롬에서만 테스트되고 있습니다.
```
Comment (weensy) [inline docs/doc_kr/README.md:44]:
```suggestion
* PyScript는 현재 크롬에서만 테스트 되었습니다.
```

File: docs/doc_kr/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## Resources
+
+* [Discussion board](https://community.anaconda.cloud/c/tech-topics/pyscript)
+* [Home Page](https://pyscript.net/)
+* [Blog Post](https://engineering.anaconda.com/2022/04/welcome-pyscript.html)
+
+## Notes
+
+* 이것은 매우 실험적인 프로젝트이므로, 프로젝트가 깨질 수 있습니다!
```
Comment (weensy) [inline docs/doc_kr/README.md:43]:
```suggestion
* 이것은 매우 실험적인 프로젝트이므로, 문제가 발생할 수 있습니다!
```

File: docs/doc_kr/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+사용방법에 대한 자세한 예시는 [pyscriptjs/examples](pyscriptjs/examples) 폴더를 확인하십시오. 크롬에서 열기만 하시면 됩니다.
+
+## Contribute
+
+[기여하기](CONTRIBUTING.md) 문서를 참조하십시오.
+
+## Resources
+
+* [Discussion board](https://community.anaconda.cloud/c/tech-topics/pyscript)
```
Comment (weensy) [inline docs/doc_kr/README.md:37]:
```suggestion
* [토론 게시판](https://community.anaconda.cloud/c/tech-topics/pyscript)
```

File: docs/doc_kr/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+* [Home Page](https://pyscript.net/)
```
Comment (weensy) [inline docs/doc_kr/README.md:38]:
```suggestion
* [홈페이지](https://pyscript.net/)
```

File: docs/doc_kr/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+* [Blog Post](https://engineering.anaconda.com/2022/04/welcome-pyscript.html)
```
Comment (weensy) [inline docs/doc_kr/README.md:39]:
```suggestion
* [블로그 포스팅](https://engineering.anaconda.com/2022/04/welcome-pyscript.html)
```

**Comment by codevilot** [comment]
@weensy Thank you for your good review. Your review makes my translation better. 

I'll do better next time. Thank you so much!!

**Comment by fpliger** [comment]
@codevilot @weensy thanks for the contribution and your kind words! It's really exciting to see!


As mentioned in #389 , we are in the process of merging the documentation infrastructure and that will follow with a push on the docs.  For this reason, I'll wait for a little to merge this just to avoid adding confusion.

@jezdez @kevingoldsmith  do you have suggestions?

**Comment by tedpatrick** [comment]
Thank you for your PR contribution to PyScript. We are closing PRs older than 90 days with limited activity.

**Pull request closed without merging**